LTSupportAutomotive/LTBTLESerialTransporter.m

```diff
@@ -231,11 +230,21 @@ -(void)peripheral:(CBPeripheral *)peripheral didDiscoverServices:(NSError *)erro
         LOG( @"Could not discover services: %@", error );
         return;
     }
+     
+    // Temp workaround to ignore pesky Bluetooth Battery Monitor
+    if ( [peripheral.name isEqualToString:@"Battery Monitor"] )
+    {
+        LOG( @"Peripheral '%@' is that darn Battery Monitor", peripheral.identifier );
+
+        [_manager cancelPeripheralConnection:peripheral];
+        [_possibleAdapters removeObject:peripheral];
+        return;
+    }
```

Comment:
This seems a bit unnecessary. I noticed you wrote "temp workaround", but still… usually you are querying for dedicated services – is that battery monitor of yours providing the same service UUID as one of your OBD2 adapters?

Reply:
Agree. I have removed this from my code now. This won't be in my next PR.